mattermost-10.0.advisories.yaml

```diff
@@ -562,6 +566,10 @@ advisories:
           note: |
             This vulnerability relates to one of mattermost's dependencies - 'github.com/mholt/archiver/v3'.
             Mattermost is running the most recent release of this dependency - v3.5, which still contains this vulnerability.
+      - timestamp: 2024-11-18T04:31:18Z
+        type: pending-upstream-fix
+        data:
+          note: The affected component mholt/archiver/v3 does not yet have a fix version, upstream maintainers must implement this.
```

Comment:
There is a fork I guess which has released a tag to use for these CVE

Reply:
https://github.com/anchore/archiver/releases/tag/v3.5.2 This is a fork of the original who has releases a new tag v3.5.2 with a [fix](https://github.com/anchore/archiver/pull/1) this CVE-2024-0406.

We can use replace in go/bump to fix this
```yaml
  - uses: go/bump
    with:
      deps: github.com/mholt/archiver/v3@v3.5.1
      replaces: github.com/mholt/archiver/v3=github.com/anchore/archiver/v3@v3.5.2
```

Similar to what [we did in multiple wolfi packages](https://github.com/search?q=repo:wolfi-dev/os+github.com/mholt/archiver/v3%3Dgithub.com/anchore/archiver/v3@v3.5.2&type=code)